CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+### Changed
+
+- Sync with upstream chart version kube-prometheus-stack-15.4.6
```

Comment:
Can you add a link to the upstream changes here?

Reply:
I could link to the comparison on github https://github.com/prometheus-community/helm-charts/compare/kube-prometheus-stack-15.2.0...kube-prometheus-stack-15.4.6

But this is not easily navigatable because they keep so many charts in the repository :(

Reply:
I see. When they don't provide a useful change log, then fine ;)